src/modelinstance.cpp

```diff
@@ -487,7 +487,7 @@ Status ModelInstance::reloadModel(const ModelConfig& config, const DynamicModelP
 Status ModelInstance::recoverFromReloadingError(const Status& status) {
     SPDLOG_WARN("Failed to reload model: {} version: {} with error: {}. Reloading to previous configuration",
         getName(), getVersion(), status.string());
-    unloadModel();
+    unloadModel(false);
```

Comment:
It may be more readable if  you put it this way:
bool changeStatus{false};
unloadModel(changeStatus);